assignments/arrays.js

```diff
@@ -74,23 +74,46 @@ console.log(`Last car is a ${inventory[lastCar].car_year} ${inventory[lastCar].c
 
 // ==== Challenge 3 ====
 // The marketing team wants the car models listed alphabetically on the website. Sort all the car model names into alphabetical order and log the results in the console
-let carModels = [];
-console.log();
+let carModels = inventory.slice();
+carModels.sort(function(a , b) {
+    let car_modelA = a.car_model.toLowerCase();
+    let car_modelB = b.car_model.toLowerCase();
+    if (car_modelA < car_modelB) {
+        return -1;
+    } if (car_modelB < car_modelA) {
+        return 1;
+    }
+})
+console.log(carModels);
```

Comment:
I do love the way you went about this although it could have been written shorter and I think you knew that, but taking a new unique and challenging approach really helps you get a deeper understanding in javascript.